frontend/src/pages/RegisterPage.jsx

```diff
@@ -1,7 +1,23 @@
+const LABEL = "block text-dark/50 font-semibold text-2xl"
+const INPUT_FIELD = "rounded-md pl-3 h-14 w-full border-solid border-2 border-dark/50 mb-4 text-lg"
+//Need to add states for inputs
```

Comment:
Can you add the states for now tho? Just make it so when you hit Register it prints the fields to the console for now